docs/reference/pages/model_reference.rst

```diff
@@ -978,9 +978,9 @@ Database fields
 
         A foreign key to the user that triggered the action.
 
-    .. attribute:: data_json
+    .. attribute:: data
```

Comment:
Probably need to add a note here somewhere to indicate the name change.

Reply:
I agree this would be good - you could pair it with a `.. versionchanged:: version` to let people know about the details of the change (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html)

Reply:
Added.